docs/source/ttnn/ttnn/api.rst

```diff
@@ -242,7 +242,6 @@ Pointwise Unary
    ttnn/log2_bw
    ttnn/sign_bw
    ttnn/fmod_bw
-   ttnn/remainder_bw
```

Comment:
any reason for removing this and adding in fw part?

Reply:
@Aswinmcw 
I moved it since I saw it was given under Pointwise Unary but it ttnn.remainder_bw is a binary overload op. I can move it back with other bw ops